src/test/java/seedu/multitasky/TestApp.java

```diff
@@ -14,16 +13,16 @@
 import seedu.multitasky.testutil.TestUtil;
 
 /**
- * This class is meant to override some properties of MainApp so that it will be suited for
- * testing
+ * This class is meant to override some properties of MainApp so that it will be
```

Comment:
@akd13 You'll need to configure Eclipse comment formatter to `never join lines`. Check out the meeting minutes.